database/postgres/index.js

```diff
@@ -1,15 +1,44 @@
 const pool = require('./db');
 
-exports.getRestaurantReviews = (restaurantId) => {
-  pool
+exports.getRestaurantReviews = ({ restaurantId }) => {
+  return pool
   .connect()
   .then(client => {
     return client
-      // same as what was being sent to db before
-      .query('SELECT  FROM users WHERE id = $1', [1])
-      .then(res => {
-        client.release()
-        console.log(res.rows[0])
+      .query(
+        `SELECT
+          restaurants.id,
+          restaurants.city,
+          reviews.id,
+          reviews.overall,
+          reviews.food,
+          reviews.service,
+          reviews.ambience,
+          reviews.dineDate,
+          reviews.noise,
+          reviews.recommend,
+          reviews.filtertag,
+          reviews.comments,
+          reviews.id,
+          reviewers.id,
+          reviewers.firstname,
+          reviewers.lastname,
+          reviewers.color,
+          reviewers.vip
+        FROM
+          restaurants,
+          reviews,
+          reviewers
```

Comment:
looks good. honestly tried to do something similar and still wasn't working out for me. nice to see you got it in one query .

Reply:
I found something that may help me and I"ll try to come back and use it if I have time.